### PR TITLE
[DOC] Add APM changes and fix links

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does / why we need it**:

Adds screenshot of the APM dashboard and fixes links from the Tempo data source page

### CODE REVIEW

File: docs/sources/datasources/tempo.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ A service graph is a visual representation of the relationships between services
 
 To display the service graph:
 
-- [Configure the Grafana Agent](https://grafana.com/docs/tempo/next/grafana-agent/service-graphs/#quickstart) or [Tempo/GET](https://grafana.com/docs/tempo/latest/server_side_metrics/service_graphs/#tempo) to generate service graph data
+- [Configure the Grafana Agent](https://grafana.com/docs/tempo/latest/grafana-agent/service-graphs/#quickstart) or [Tempo/GET](https://grafana.com/docs/tempo/latest/server_side_metrics/service_graphs/#tempo) to generate service graph data
```
Comment (brendamuir) [inline docs/sources/datasources/tempo.md:185]: Hey Kim, all looks great -- just one thing from my short time on Agent docs -- we don't use an article in front of agent -- so it's just Configure Grafana Agent.

File: docs/sources/datasources/tempo.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ weight: 1400
 
 # Tempo data source
 
-Grafana ships with built-in support for Tempo a high volume, minimal dependency trace storage, OSS tracing solution from Grafana Labs. Add it as a data source, and you are ready to query your traces in [Explore]({{< relref "../explore/" >}}).
+Grafana ships with built-in support for Tempo, a high volume, minimal dependency trace storage, OSS tracing solution from Grafana Labs. Add it as a data source, and you are ready to query your traces in [Explore]({{< relref "../explore/" >}}).
```
Comment (Eve832) [inline docs/sources/datasources/tempo.md:18]:
```suggestion
Grafana ships with built-in support for Tempo, a high volume, minimal dependency trace storage, OSS tracing solution from Grafana Labs. Add it as a data source, and you are ready to query your traces in [Explore]({{< relref "../explore" >}}).
```

**Review by Eve832 — CHANGES_REQUESTED** [review]
looks good, with one edit

File: docs/sources/explore/trace-integration.md [hunk 3]
```diff
@@ -22,23 +22,27 @@ Supported data sources are:
 
 For information on how to configure queries for the data sources listed above, refer to the documentation for specific data source.
 
+## Trace View explanation
+
+This section explains the elements of the Trace View dashboard.
```
Comment (knylander-grafana) [inline docs/sources/explore/trace-integration.md:27]:
```suggestion
This section explains the elements of the Trace View dashboard.
```

File: docs/sources/datasources/tempo.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/datasources/tempo.md]: Committed the change. Thank you.

**Review by Eve832 — APPROVED** [review]
lgtm!

**Pull request merged**